controller.go

```diff
@@ -1255,7 +1256,7 @@ func (c *controller) loadDriver(networkType string) error {
 	}
 
 	if err != nil {
-		if err == plugins.ErrNotFound {
+		if errors.Cause(err) == plugins.ErrNotFound {
```

Comment:
This fixes:

```
--- FAIL: TestUnknownDriver (15.00s)
	libnetwork_test.go:213: Did not fail with expected error. Actual error: could not find plugin unknowndriver in v1 plugin registry: plugin not found

--- FAIL: TestNilRemoteDriver (15.00s)
	libnetwork_test.go:225: Did not fail with expected error. Actual error: could not find plugin framerelay in v1 plugin registry: plugin not found
```

Reply:
Wonder if this is an actual issue currently; this was revealed due to docker/docker being bumped here, but likely is an issue in the current version in that repository